regexes/client/mobile_apps.yml

```diff
@@ -1036,3 +1036,23 @@
 - regex: 'imoAndroid/(\d+\.[\.\d]+)'
   name: 'IMO International Calls & Chat'
   version: '$1'
+
+# Microsoft Bing Search (https://play.google.com/store/apps/details?id=com.microsoft.bing | https://apps.apple.com/us/app/microsoft-bing-search/id345323231)
+- regex: 'BingSapphire/(\d+\.[\.\d]+)'
+  name: 'Microsoft Bing Search'
+  version: '$1'
+
+# HiSearch
+- regex: '.+HiSearch/(\d+\.[\.\d]+)'
+  name: 'HiSearch'
+  version: '$1'
+
+# RDDocuments
+- regex: 'RDDocuments/(\d+\.[\.\d]+)'
+  name: 'RDDocuments'
+  version: '$1'
```

Comment:
it seems that this user agent refers to the program

add link to app in  comment  https://apps.apple.com/app/id364901807

https://readdle.com/documents
https://www.webmasterworld.com/search_engine_spiders/4961327.htm